### ISSUE TITLE
Trinity compute: docker fails to start

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#176]
**version:** 9f2fae3

```
[root@node002 ~]# systemctl status docker
docker.service - Docker Application Container Engine
   Loaded: loaded (/usr/lib/systemd/system/docker.service; enabled)
  Drop-In: /usr/lib/systemd/system/docker.service.d
           └─remote-fs.conf, ulimit.conf
   Active: inactive (dead)
     Docs: http://docs.docker.com

apr 10 21:09:57 node002.cluster systemd[1]: Dependency failed for Docker Application Container Engine.
```

**Comment by hansthen** [comment]
```
[root@node002 ~]# journalctl -xn
-- Logs begin at vr 2015-04-10 21:09:55 CEST, end at vr 2015-04-10 21:22:31 CEST. --
apr 10 21:21:45 node002.cluster sshd[1641]: pam_unix(sshd:session): session opened for user root by (uid=0)
apr 10 21:22:31 node002.cluster systemd[1]: Mounting /cm/shared...
-- Subject: Unit cm-shared.mount has begun with start-up
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- 
-- Unit cm-shared.mount has begun starting up.
apr 10 21:22:31 node002.cluster mount[1669]: mount.nfs: mounting controller:/home/cm/shared failed, reason given by server: No such file or directory
apr 10 21:22:31 node002.cluster systemd[1]: cm-shared.mount mount process exited, code=exited status=32
apr 10 21:22:31 node002.cluster systemd[1]: Failed to mount /cm/shared.
-- Subject: Unit cm-shared.mount has failed
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- 
-- Unit cm-shared.mount has failed.
-- 
-- The result is failed.
apr 10 21:22:31 node002.cluster systemd[1]: Dependency failed for Remote File Systems.
-- Subject: Unit remote-fs.target has failed
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- 
-- Unit remote-fs.target has failed.
-- 
-- The result is dependency.
apr 10 21:22:31 node002.cluster systemd[1]: Dependency failed for Docker Application Container Engine.
-- Subject: Unit docker.service has failed
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- 
-- Unit docker.service has failed.
-- 
-- The result is dependency.
apr 10 21:22:31 node002.cluster systemd[1]: Unit cm-shared.mount entered failed state.
```

**Comment by hansthen** [comment]
/etc/fstab
```
proc            /proc    proc   rw 0 0
sysfs           /sys     sysfs  rw 0 0
devpts          /dev/pts devpts rw,gid=5,mode=620 0 0
controller:/home           /home    nfs    rw  0 0
controller:/home/cm/shared  /cm/shared  nfs  rw 0 0
/dev/sda1 /var/lib/docker ext4 rw 0 0
controller:/cluster        /cluster    nfs    rw  0 0
```

**Comment by hansthen** [comment]
There was a stale /etc/fstab file in the /trinity/compute/rootimg directory.

**Comment by hansthen** [comment]
**Fixed in:** 4bcce3c
